### ISSUE TITLE
Report offset and margin or paragraphs in word pad

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#12468]
### Steps to reproduce:
open word pade.
open the paragraph window, you can:
press alt+h, then p, then g.
Language Portuguese Brazil, I don't know if changing the language the shortcuts change.
In the paragraph window, tab and in the first line edit box fill in 1 will become 1 cm.
Tab to ok and press enter.### Actual behavior:
when pressing nvda + f on the text that received the formatting, the information about paragraph displacement of the left margin of a few centimeters is not read. 
### Expected behavior:
information about paragraph offset of left margin should be read 
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
nvda.exe, NVDA alpha-22934,e05715d3

#### Windows version:
10 19043.1021
#### Name and version of other software in use when reproducing the issue:
word pade
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
[frases bonitas 2.rtf.txt](https://github.com/nvaccess/nvda/files/6549728/frases.bonitas.2.rtf.txt)

remove the .txt extension from the end of the file

**Comment by Adriani90** [comment]
I guess this needs to be reported to Microsoft, this information is not exposed when creating a list with different levels in Wordpad or Notepad using tab or space bar.
NVDA cannot do anything about this I guess.

cc: @CyrilleB79

**Comment by CyrilleB79** [comment]
@fernando-jose-silva it's not clear to me what you refer to. Are you referring to paragraph indent?
Could you update the initial description with more detailed steps to reproduce, e.g. a bullet list.

@Adriani90 paragraph indent may be reported by Narrator in WordPad. So the information may be available in some way. Maybe using [EM_GETPARAFORMAT](https://learn.microsoft.com/en-us/windows/win32/controls/em-getparaformat) message?

**Comment by Adriani90** [comment]
Wordpad is going to be removed in Windows 11 completely during this year, it is a dicontinued application. i don't expect any further accessibility updates to this application, but let's keep this open until it is officially removed from Windows.
See this article for more details.
https://www.tweaktown.com/news/95854/windows-11-users-wave-goodbye-to-wordpad-itll-likely-be-removed-with-the-24h2-update/index.html

**Comment by josephsl** [comment]
Hi,

April 2024 update: yes, WordPad will be removed in Windows 11 2024 Update, so let's vote to close this (for now).

Thanks.

**Comment by CyrilleB79** [comment]
Note that Rich edit text, may be used elsewhere, e.g. "Other details" in add-on store or any wx application.
I understand that this dev becomes very low priority but I would not have closed it anyway. And overall, not until NVDA only supports Windows 11 24h2 or higher; for now it still supports Windows 8.1 and Windows 10 which embed Wordpad.

@josephsl I'd better let you the final decision for now, but I'd highly recommend to reopen.

Also, if anybody else needs this feature, please comment here and we can reopen.
